DataFormats/PatCandidates/interface/CovarianceParameterization.h

```diff
@@ -0,0 +1,54 @@
+#ifndef _DataFormats_PatCandidates_CovarianceParameterization_h_
+#define _DataFormats_PatCandidates_CovarianceParameterization_h_
+#include <TFile.h>
+#include <TH3D.h>
+#include <iostream>
+#include <unordered_map>
+#include <TKey.h>
+class CompressionElement {
+    public:
+      enum Method {float16=0,reduceMantissa=1,logPack=2,tanLogPack=3,zero=4,one=5};
+      enum Target {realValue=0,ratioToRef=1,differenceToRef=2};
+      CompressionElement():method(zero),target(realValue){}
+      //CompressionElement(Method m, Target t, int bitsUsed, std::initializer_list<float> p): method(m),target(t),bits(bitsUsed),params(p){}
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant